### PR TITLE
updated homepage to match the redlines

### PR DESCRIPTION
**Pull request by RiehaB**
Signed-off-by: Rieha <rieha.burrell@ibm.com>

### CODE REVIEW

File: client/templates/HomeView.ejs [hunk 1]
```diff
@@ -127,6 +128,14 @@
             margin-right:32px;
         }
 
+        #content-middle {
+            margin-bottom: 3px;
```
Comment (cazfletch) [inline client/templates/HomeView.ejs]: this should be either 2px or 4px so we always size things as 2, 4, 8, 12, 16, 32 or 64

File: client/templates/HomeView.ejs [hunk 2]
```diff
@@ -147,7 +156,9 @@
             border-radius: 5px;
             display: flex;
             flex-direction: column;
-            height: 100%;
+            min-height: 250px;
+            padding: 16px;
+            padding-right: 90px;
```
Comment (cazfletch) [inline client/templates/HomeView.ejs]: this doesn't look right and suggests the css is wrong somewhere

File: client/package.json [hunk 3]
```diff
@@ -33,7 +33,39 @@
         "Other"
     ],
     "activationEvents": [
-        "*"
+        "onView:gatewayExplorer",
```
Comment (cazfletch) [inline client/package.json]: This should be set to *

File: client/templates/HomeView.ejs [hunk 4]
```diff
@@ -127,6 +128,14 @@
             margin-right:32px;
         }
 
+        #content-middle {
+            margin-bottom: 2px;
+        }
+
+        #content-top-right {
+            margin-top: 22px;
```
Comment (cazfletch) [inline client/templates/HomeView.ejs]: This should be 24px as needs to follow the rules of having spacings to be one of 2px, 4px, 8px, 16px, 24px, 32px, and 64px

**Comment by Jakeeyturner** [comment]
@RiehaB is out this week, so I'll pick up these changes and get this completed (for if we decide to to a release this week).

**Comment by Jakeeyturner** [comment]
Closing as PR #850 covers this!

**Pull request closed without merging**